jobbergate-cli/jobbergate_cli/subapps/job_scripts/tools.py

```diff
@@ -386,3 +386,14 @@ def download_job_script_files(id: int, jg_ctx: JobbergateContext) -> List[pathli
         destination_path=pathlib.Path.cwd(),
     )
     return downloaded_files
+
+
+def question_helper(question_func: Callable, text: str, default: bool, fast: bool, actual_value: Optional[bool]):
+    """
+    Helper function for asking questions to the user.
```

Comment:
Could you document the priority order here at the docstring?

Reply:
I improved the docstring for this method.

Reply:
That looks great